tasks/main.yml

```diff
@@ -5,12 +5,12 @@
   apt_repository: repo="{{ tsuru_repo }}" update_cache=yes
 
 - name: Install packages.
-  apt: update_cache=true name="{{ item }}"
+  apt: update_cache=true name="{{ item }}" "{% if tsuru_package_latest is defined %}state=latest{% else %}state=present{% endif %}"
   with_items:
     - python-software-properties
-    - "{% if tsuru_server_version is defined %}tsuru-server={{ tsuru_server_version }}{% else %}tsuru-server{% endif %}"
-    - "{% if tsuru_admin_version is defined %}tsuru-admin={{ tsuru_admin_version }}{% else %}tsuru-admin{% endif %}"
-    - "{% if tsuru_client_version is defined %}tsuru-client={{ tsuru_client_version }}{% else %}tsuru-client{% endif %}"
+    - "{% if tsuru_server_version is defined and tsuru_package_latest is not defined %}tsuru-server={{ tsuru_server_version }}{% else %}tsuru-server{% endif %}"
```

Comment:
(I realise that's not strictly part of this PR but it seems like a good opportunity as we're adding more conditions to it)